src/gui/properties/propertieswidget.cpp

```diff
@@ -108,6 +108,8 @@ PropertiesWidget::PropertiesWidget(QWidget *parent)
             , m_ui->filesList, qOverload<const QModelIndex &>(&QAbstractItemView::edit));
     connect(m_ui->filesList, &QWidget::customContextMenuRequested, this, &PropertiesWidget::displayFilesListMenu);
     connect(m_ui->filesList, &QAbstractItemView::doubleClicked, this, &PropertiesWidget::openItem);
+    m_ui->filesList->header()->setContextMenuPolicy(Qt::CustomContextMenu);
```

Comment:
Find a better place for this line.

Reply:
[Here](https://github.com/qbittorrent/qBittorrent/blob/bd1582ed542f55d531190d92620a1d105af9d90e/src/gui/properties/propertieswidget.cpp#L84-L85)?